src/test/java/pl/put/poznan/transformer/logic/serviceimpl/ScenarioNumberedListServiceImplTest.java

```diff
@@ -0,0 +1,122 @@
+package pl.put.poznan.transformer.logic.serviceimpl;
+
+import org.junit.Before;
+import org.junit.Test;
+import pl.put.poznan.transformer.logic.domain.dto.ScenarioDTO;
+import pl.put.poznan.transformer.logic.domain.dto.ScenarioStepDTO;
+import pl.put.poznan.transformer.logic.service.ScenarioNumberedListService;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+public class ScenarioNumberedListServiceImplTest {
+    private ScenarioNumberedListService scenarioNumberedListService;
+
+    private final String EXAMPLE_TITLE = "HOLIDAYS";
+    private final String EXAMPLE_ACTOR = "EMPLOYEE";
+    private final String EXAMPLE_SYSTEM_ACTOR = "SOFTWARE";
+    private final String EXAMPLE__STEP_NAME = "EMPLOYEE REQUESTS HOLIDAYS";
+
+    @Before
+    public void setUp() {
+        this.scenarioNumberedListService = new ScenarioNumberedListServiceImpl();
+    }
+
+
+    @Test
+    public void testGetScenarioAsNumberedListReturnOnlyHeaderOnEmptyScenario() {
+        ScenarioDTO scenarioDTO = new ScenarioDTO();
+        String result = scenarioNumberedListService.getScenarioAsNumberedList(scenarioDTO);
+        assertTrue(result.startsWith("Title:"));
+        assertFalse(result.contains("1. "));
+    }
+
+    @Test
+    public void testGetScenarioAsNumberedListReturnCorrectValueWithOneDepthScenario() {
+        ScenarioDTO scenarioDTO = prepareOneDepthScenarioWithThreeSteps();
+        String result = scenarioNumberedListService.getScenarioAsNumberedList(scenarioDTO);
+        assertTrue(result.startsWith(String.format("Title: %s", EXAMPLE_TITLE)));
+
+        assertTrue(result.contains(String.format("1. %s", EXAMPLE__STEP_NAME)));
+        assertTrue(result.contains(String.format("2. %s", EXAMPLE__STEP_NAME)));
+        assertTrue(result.contains(String.format("3. %s", EXAMPLE__STEP_NAME)));
+        assertFalse(result.contains("1.1 "));
+
+    }
+
+    @Test
+    public void testGetScenarioAsNumberedListReturnCorrectValueWithMixedThreeDepthScenario() {
+        ScenarioDTO scenarioDTO = prepareThreeDepthsMixedScenario();
+        String result = scenarioNumberedListService.getScenarioAsNumberedList(scenarioDTO);
+        assertTrue(result.startsWith(String.format("Title: %s", EXAMPLE_TITLE)));
+
```

Comment:
Assertion is an one block logic so it shouldn't be separated

Reply:
Sorry I'm not sure what to change here.

Reply:
Code in function is often organized in logical blocks with use of newlines. You have two such blocks here. What I tried to say is that one assertion is separated from others like if it was special and didn't belong to logic realized by other assertions below. It seems to be not a big deal but often the small things make a big difference, or something like that :)